go.mod

```diff
@@ -0,0 +1,15 @@
+module htop-clone
+
+go 1.18
+
+require github.com/shirou/gopsutil/v3 v3.22.6
+
+require (
+	github.com/go-ole/go-ole v1.2.6 // indirect
+	github.com/lufia/plan9stats v0.0.0-20211012122336-39d0f177ccd0 // indirect
```

Comment:
remember to run `go mod tidy` to clean stuff you don't end up using.

Reply:
I did and it still brings those indirect requirements. I think it is because the package itself needs it like shown [here](https://github.com/shirou/gopsutil/blob/master/go.mod).